GoogleDataTransport/GDTCORLibrary/GDTCORStorage.m

```diff
@@ -114,15 +114,17 @@ - (void)storeEvent:(GDTCOREvent *)event
                                         eventHash:event.hash
                                             error:&error];
     GDTCORLogDebug("Event saved to disk: %@", eventFile);
-    GDTCORDataFuture *dataFuture = [[GDTCORDataFuture alloc] initWithFileURL:eventFile];
-    GDTCORStoredEvent *storedEvent = [event storedEventWithDataFuture:dataFuture];
+    [event setFileURL:eventFile];
```

Comment:
Change all setFileURL: calls to the more modern dot syntax: `event.fileURL = eventFile;`